testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/cli/OptionsPrinter.java

```diff
@@ -106,6 +108,17 @@ else if (annotation.negatable()) {
             }
         }
 
+        if (value instanceof Map) {
+            if (((Map<?, ?>) value).isEmpty()) {
+                return null;
+            }
+            else {
```

Comment:
redundant else

but, isn't the `if` redundant as well? (is `""` treated differently than `null` by the caller?)

Reply:
Empty string causes an empty line with `\` at the end. `null` suppresses output for the line entirely.

Will fix the redundant else though.